tests/osaka/eip7692_eof_v1/eip4750_functions/test_code_validation.py

```diff
@@ -572,6 +572,77 @@ def test_callf_stack_overflow_by_height(eof_test, callee_stack_height):
     eof_test(container=container, expect_exception=EOFException.STACK_OVERFLOW)
 
 
+@pytest.mark.parametrize(
+    "container",
+    [
+        Container(
+            name="underflow_1",
+            sections=[
+                Section.Code(
+                    code=Op.CALLF[1] + Op.STOP,
+                    max_stack_height=1,
+                ),
+                Section.Code(
+                    code=Op.PUSH0 + Op.RETF,
+                    code_inputs=1,
+                    code_outputs=2,
+                    max_stack_height=2,
+                ),
+            ],
+            expected_bytecode="ef000101000802000200040002040000000080000101020002e30001005fe4",
+        ),
+        Container(
+            name="underflow_variable_stack_2",
+            sections=[
+                Section.Code(
+                    code=Op.PUSH0
+                    + Op.PUSH1[0]
```

Comment:
Just wanted to confirm that the purpose of this test is to have an underflow in both branches of the `RJUMPI`, in contrast to the one below where only one of the branches has the underflow.